### ISSUE TITLE
How to directly obtain the entire testset?

### ISSUE THREAD

**Issue by Lewis-lv0** [Junchao-cs/SolarWM#7]
Hi @Junchao-cs , thank you for releasing this amazing project. I would like to download the entire testset to see its performance.

However, assembling the standalone test set from the published raw-WDS shards requires downloading 1,196 primary shards / ~1.16 TB to extract roughly 82 GB of actual test assets. I wonder if there is any more convenient ways for us to just download the testset without extra training data.

I would appreciate your assistance on this.

**Comment by Junchao-cs** [comment]
Hi, thank you for your interest in SolarWM!

We have now released a standalone version of the test set, so you no longer need to download the full training dataset.

https://huggingface.co/datasets/junchaoh-cs/SolarWM-Data_test-set-v1

We hope this makes evaluation much more convenient!
